packages/@react-spectrum/s2/src/Content.tsx

```diff
@@ -102,17 +103,26 @@ export const TextContext = createContext<ContextValue<ContentProps, DOMRefValue>
 function Text(props: ContentProps, ref: DOMRef) {
   [props, ref] = useSpectrumContextProps(props, ref, TextContext);
   let domRef = useDOMRef(ref);
-  let {UNSAFE_className = '', UNSAFE_style, styles, isHidden, slot, ...otherProps} = props;
+  let {UNSAFE_className = '', UNSAFE_style, styles, isHidden, slot, children, ...otherProps} = props;
+  let racContext = useContext(RACTextContext);
+  let isSkeleton = useIsSkeleton();
+  [children, UNSAFE_style] = useSkeletonText(children, UNSAFE_style);
   if (isHidden) {
     return null;
   }
+
+  slot = slot && racContext && 'slots' in racContext && !racContext.slots?.[slot] ? undefined : slot;
```

Comment:
Not sure if we want this or not. The idea is to not forward Spectrum-specific slots that RAC doesn't expect

Reply:
Seems related: https://github.com/adobe/react-spectrum/issues/6992